packages/application-shell/src/components/application-page-title/application-page-title.tsx

```diff
@@ -0,0 +1,127 @@
+import type { TEnhancedLocation } from '@commercetools-frontend/browser-history';
+
+import { useLayoutEffect } from 'react';
+import { useLocation } from 'react-router-dom';
+import startCase from 'lodash/startCase';
+import capitalize from 'lodash/capitalize';
+
+type Breadcrumb = {
+  suffix: string;
+  paths: string[];
+};
+type Props = {
+  /**
+   * Allow to render a custom page title.
+   * Overrides default page title with the format:
+   * <?resource_name(manually inputed)> - <?page_location> - <application_identifier> - <project_key> - Merchant Center
+   */
+  renderPageTitle?: string[];
```

Comment:
Nit: the name of this prop seems a bit confusing to me. In the RFC I proposed `content` but we can also find another name as long as it fits with what it does.